### PR TITLE
Update README. Remove/simplify repo contents

### PR DESCRIPTION
**Pull request by sergio-opslevel**
Issue: #6042

## Summary

* Update README w/ feedback from the team & GTM
* Link repository in package.json. Hopefully now it will appear on [npmjs.org](https://www.npmjs.com/package/backstage-plugin-opslevel-maturity)
* Simplify `tsconfig.json`
* Remove unused `dev/`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,9 +67,9 @@ import CheckCircleOutlineIcon from '@material-ui/icons/CheckCircleOutline';
 ```
 
 
-## Add Entity tab for Maturity
+## Add Tab for Maturity to Services
 
-Add to the `serviceEntityPage` in `packages/app/src/components/catalog/EntityPage.tsx`
+In `packages/app/src/components/catalog/EntityPage.tsx` import the plugin and add it to `serviceEntityPage`
```
Comment (sergio-opslevel) [inline README.md:72]: I found I often misread this line. The established pattern from the previous instructions was filename first, so I updated this to match.

File: README.md [hunk 2]
```diff
@@ -68,3 +83,8 @@ import { EntityOpsLevelMaturityContent } from 'backstage-plugin-opslevel-maturit
 ## Next Steps
 
 Visit the Maturity tab in Backstage to get started.
+
+
+### Maturity of this Plugin
+
+[![Overall](https://img.shields.io/endpoint?style=flat&url=https%3A%2F%2Fapp.opslevel.com%2Fapi%2Fservice_level%2FSX_5tBBV3PXtQcTEe4j6kGw_Sm0ys-piO0swtoWKCwo)](https://app.opslevel.com/services/backstage_plugin/maturity-report)
```
Comment (sergio-opslevel) [inline README.md:90]: To pass [our check](https://app.opslevel.com/services/backstage_plugin/maturity-report?checkId=Z2lkOi8vb3BzbGV2ZWwvQ2hlY2tzOjpSZXBvRmlsZS82ODU2). Platinum here we come!

File: tsconfig.json [hunk 3]
```diff
@@ -1,45 +1,10 @@
 {
+  "extends": "@backstage/cli/config/tsconfig.json",
   "compilerOptions": {
-    "allowJs": true,
```
Comment (sergio-opslevel) [inline tsconfig.json:3]: The file I'm extending: https://github.com/backstage/backstage/blob/master/packages/cli/config/tsconfig.json

File: README.md [hunk 4]
```diff
@@ -5,10 +5,25 @@ This plugin helps you leverage OpsLevel's powerful maturity features with your e
...
 ![](docs/maturity.jpg)
 
+[OpsLevel](https://www.opslevel.com/) gives platform engineers and DevOps leaders the tools to drive service maturity for their teams. We understand that maturity features alongside your software catalog are critical to a culture of service ownership. Which is why we’ve made our maturity features available to Backstage users within their existing service catalog.
+
+OpsLevel uses [Checks](https://www.opslevel.com/docs/getting-started-with-checks) to measure Service Maturity. Checks let you explicitly define how you want your services to be built and operated.
+With OpsLevel Checks, you can verify that services:
+* Are using a particular version of a library or framework
+* Have migrated to a new third-party tool (e.g., all services use Splunk)
+* Meet certain operational requirements (e.g., all Tier-1 services have an on-call schedule)
+And a whole lot more.
```
Comment (sabeerzaman) [inline README.md:15]: Need whitespace so that this shows up in a newline and not part of the last bullet.

```suggestion

And a whole lot more.
```

**Review by sabeerzaman — APPROVED** [review]
Had just the one nit to help fix Markdown formatting. LGTM otherwise!

**Pull request merged**